src/MetaMorpho.sol

```diff
@@ -794,30 +820,6 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
         return (assets.zeroFloorSub(idle), idle.zeroFloorSub(assets));
     }
 
-    /// @dev Returns the suppliable amount of assets on the market defined by `marketParams`.
-    /// @dev Assumes that the inputs `marketParams` and `id` match.
-    function _suppliable(MarketParams memory marketParams, Id id) internal view returns (uint256) {
-        uint256 supplyCap = config[id].cap;
-        if (supplyCap == 0) return 0;
-
-        return supplyCap.zeroFloorSub(_supplyBalance(marketParams));
-    }
-
-    /// @dev Returns the withdrawable amount of assets from the market defined by `marketParams`.
-    /// @dev Assumes that the inputs `marketParams` and `id` match.
-    function _withdrawable(MarketParams memory marketParams, Id id) internal view returns (uint256) {
```

Comment:
I would keep this function, we're duplicating code now

Reply:
We're not duplicating code: `_withdrawable` was used once in `_simulateWithdrawMorpho` virtually accruing interest and once in `_withdrawMorpho` which now accrues interest on Morpho directly

Reply:
I factorized it, wdyt?

Reply:
it's better to me thx